src/css/base/utility-classes.scss

```diff
@@ -0,0 +1,1767 @@
+@import '../core.scss';
+
+// base utility classes, primarily for documentation and prototyping
+
+$u-percentages: '5', '10', '20', '25', '30', '33', '40', '50', '60', '66', '70', '75', '80', '90', '100';
+$u-percentage-suffix: 'ct';
+$u-percentages-s: '5%', '10%', '20%', '25%', '30%', '33.3333333333%', '40%', '50%', '60%', '66.6666666666', '70%', '75%', '80%', '90%', '100%';
+$u-pixels: '1px', '2px';
+$u-gridscale: '05', '1', '105', '2', '3', '4', '5', '6', '7', '8', '9', '10', '15', '20', '25', '30', '40', '50';
+$grid-s: $grid-05, $grid-1, $grid-105, $grid-2, $grid-3, $grid-4, $grid-5, $grid-6, $grid-7, $grid-8, $grid-9, $grid-10, $grid-15, $grid-20, $grid-25, $grid-30, $grid-40, $grid-50;
+$grid-n-multipliers: '05', '1', '105', '2', '3', '4', '5';
+$grid-n-s: $grid-n05, $grid-n1, $grid-n105, $grid-n2, $grid-n3, $grid-n4, $grid-n5;
+
+/* stylelint-disable declaration-no-important  */
+
+/*
+
+  COLORS
+
+  Base:
+    ---
+
+  Modifiers:
+    ---
+
+  Values:
+    {$color}
+
+*/
+
+$color-s: 'blue', 'brightmint', 'cream', 'gold', 'green', 'red', 'trueblack', 'textblack', 'darkgray', 'mediumgray', 'mediumgray-trans', 'lightgray', 'lightestgray', 'white';
+$colorvar-s: $color-blue, $color-brightmint, $color-cream, $color-gold, $color-green, $color-red, $color-trueblack, $color-textblack, $color-darkgray, $color-mediumgray, $color-mediumgray-trans, $color-lightgray, $color-lightestgray, $color-white;
```

Comment:
i will look up how to to do this. tho does it need to be a part of this PR — why isn't this a nice to have that we can add later?? :(